styles/site-styles.css

```diff
@@ -0,0 +1,164 @@
+
+@font-face {
+    font-family: gotham-light;
+    src: url(./Gotham-Font/Gotham-Light.otf) url(./Gotham-Font/GothamLight.ttf);
+}
+
+@font-face {
+    font-family: gotham-black;
+    src: url(./Gotham-Font/Gotham-Black.otf) url(./Gotham-Font/GothamBold.ttf);
+}
+
+body {
+    margin: auto;
+    color: white;
+    background-color: rgb(216,192,208);
+    font-family: gotham-light;
+}
+
+header {
+    display: flex;
+    height: 96px;
+    justify-content: space-around;
+    align-items: center;
+    color: rgb(232,156,44);
+    background-image: url(../images/headerphoto.jpg);
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: top;
+    padding: 50px 125px;
+}
+
+nav {
+    display: flex;
+    font-size: 1em;
+    justify-content: space-between;
+}
+
+a {
+    font: 1em;
+    padding: 20px;
+}
+
+.logo {
+    width: 60px;
+    height: 60px;
```

Comment:
i noticed your logo text isn't in line with the nav text. main reason is you've set the height. remove the height and flex box will center it for you `align-items: center;`